pandas/core/groupby/ops.py

```diff
@@ -799,14 +799,37 @@ def apply(
         # This calls DataSplitter.__iter__
         zipped = zip(group_keys, splitter)
 
+        i = 0
```

Comment:
This `i` variable doesn't look to be used anywhere

Reply:
I have to remove it, it was created for my old code.

Reply:
Same here